Common/include/option_structure.hpp

```diff
@@ -2243,15 +2243,15 @@ static const MapType<string, ENUM_VERIFICATION_SOLUTIONS> Verification_Solution_
 /*!
  * \brief Types of streamwise periodicity.
  */
-enum ENUM_STREAMWISE_PERIODIC {
-  NO_STREAMWISE_PERIODIC = 0, /*!< \brief No streamwise periodic flow. */
-  PRESSURE_DROP          = 1, /*!< \brief Prescribed pressure drop. */
-  STREAMWISE_MASSFLOW    = 2, /*!< \brief Prescribed massflow. */
+enum class ENUM_STREAMWISE_PERIODIC {
+  NONE,          /*!< \brief No streamwise periodic flow. */
+  PRESSURE_DROP, /*!< \brief Prescribed pressure drop. */
+  MASSFLOW,      /*!< \brief Prescribed massflow. */
 };
 static const MapType<string, ENUM_STREAMWISE_PERIODIC> Streamwise_Periodic_Map = {
-  MakePair("NONE",          NO_STREAMWISE_PERIODIC)
-  MakePair("PRESSURE_DROP", PRESSURE_DROP)
-  MakePair("MASSFLOW",      STREAMWISE_MASSFLOW)
+  MakePair("NONE",          ENUM_STREAMWISE_PERIODIC::NONE)
+  MakePair("PRESSURE_DROP", ENUM_STREAMWISE_PERIODIC::PRESSURE_DROP)
+  MakePair("MASSFLOW",      ENUM_STREAMWISE_PERIODIC::MASSFLOW)
```

Comment:
It ends up being a little more verbose but you get to use "names" that would otherwise conflict with what is already in use for objective functions and things like that.